UITests/LandscapeToolbarTests.swift

```diff
@@ -0,0 +1,89 @@
+
+        tester().tapViewWithAccessibilityIdentifier("url")
+        textView = tester().waitForViewWithAccessibilityLabel("Address and Search") as? UITextField
+        XCTAssertEqual(textView!.text, url1, "Text is url")
+
+        let url2 = "\(webRoot)/numberedPage.html?page=2"
+        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url2)\n")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 2")
+
+        tester().tapViewWithAccessibilityLabel("Back")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Open new tab and then go back to previous tab to test navigation buttons.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        var addTabButton = tester().waitForViewWithAccessibilityLabel("Add Tab") as? UIButton
+        addTabButton?.tap()
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().tapViewWithAccessibilityLabel("Page 1")
+        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+
+        // Test to see if all the buttons are enabled.
+        tester().waitForTappableViewWithAccessibilityLabel("Back")
+        tester().waitForTappableViewWithAccessibilityLabel("Forward")
+        tester().waitForTappableViewWithAccessibilityLabel("Reload")
+        tester().waitForTappableViewWithAccessibilityLabel("Share")
+        tester().waitForTappableViewWithAccessibilityLabel("Bookmark")
+
+        // Go Back to other tab to see if all buttons are disabled.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().tapViewWithAccessibilityLabel("home")
+        tester().waitForViewWithAccessibilityLabel("Web content")
+
+        var back = tester().waitForViewWithAccessibilityLabel("Back") as? UIButton
+        var forward = tester().waitForViewWithAccessibilityLabel("Forward") as? UIButton
+        var reload = tester().waitForViewWithAccessibilityLabel("Reload") as? UIButton
+        var share = tester().waitForViewWithAccessibilityLabel("Share") as? UIButton
+        var bookmark = tester().waitForViewWithAccessibilityLabel("Bookmark") as? UIButton
+        
+        XCTAssertFalse(back?.enabled ?? true, "Back button should be disabled")
+        XCTAssertFalse(forward?.enabled ?? true, "Forward button should be disabled")
+        XCTAssertFalse(reload?.enabled ?? true, "Reload button should be disabled")
+        XCTAssertFalse(share?.enabled ?? true, "Share button should be disabled")
+        XCTAssertFalse(bookmark?.enabled ?? true, "Bookmark button should be disabled")
+
+        // Closes tabs.
+        tester().tapViewWithAccessibilityLabel("Show Tabs")
+        tester().swipeViewWithAccessibilityLabel("Page 1", inDirection: KIFSwipeDirection.Left)
+        tester().tapViewWithAccessibilityLabel("home")
+        tester().waitForViewWithAccessibilityLabel("Web content")
```

Comment:
Is this chunk something you're testing, or is it just cleanup? Seems like it's just cleanup, in which case you should be able to remove this block (this should all be taken care of for you in `BrowserUtils.resetToAboutHome(tester())`).